### ISSUE TITLE
Selecting a named instance

### ISSUE THREAD

**Issue by matthiasclasen** [harfbuzz/harfbuzz#1864]
It would be nice to have an api to select a named instance of a hb_face_t.

**Comment by behdad** [comment]
Currently we have:
```
HB_EXTERN unsigned int
hb_ot_var_named_instance_get_design_coords (hb_face_t    *face,
                                            unsigned int  instance_index,
                                            unsigned int *coords_length, /* IN/OUT */
                                            float        *coords         /* OUT */);
```
and
```
HB_EXTERN void
hb_font_set_var_coords_design (hb_font_t *font,
                               const float *coords,
                               unsigned int coords_length);
```

So we should add `hb_font_set_var_named_instance (hb_font_t *font, unsigned instance_index)`.

**Comment by ebraminio** [comment]
A fix uploaded on #1873
